docs/installation.md

```diff
@@ -21,45 +14,59 @@ If you are using [`ghcup`](https://www.haskell.org/ghcup/) to manage your instal
 ghcup install hls
 ```
 
-You can check if HLS is available for your platorm via ghcup here: <https://haskell.org/ghcup/install/#supported-platforms>.
+You can check if HLS is available for your platform via `ghcup` here: <https://haskell.org/ghcup/install/#supported-platforms>.
 
 You can also install HLS from source without checking out the code manually:
 
+```bash
+# `ghcup compile` uses cabal under the hood to build, so you may want to run `cabal update` beforehand
+cabal update
+ghcup compile hls -v $HLS_VERSION --ghc $GHC_VERSION
+```
+
+For example:
+
+Install HLS 1.6.1.0 for GHC 8.10.7:
 ```bash
 ghcup compile hls -v 1.6.1.0 --ghc 8.10.7
 ```
 
-Install HLS for multiple GHC versions:
+Install HLS 1.7.0.0 for multiple GHC versions:
+```
+ghcup compile hls -v 1.7.0.0 --ghc 8.10.7 --ghc 8.8.4
+```
+
+Install HLS from master:
 ```
-ghcup compile hls -v 1.6.1.0 --ghc 8.10.7 --ghc 8.8.4
+ghcup compile hls -g master --ghc 9.2.1
 ```
 
 Use a different `cabal.project` for a GHC version:
 ```
-ghcup compile hls -v 1.6.1.0 --ghc 9.2.1 --cabal-project cabal.project
+ghcup compile hls -v 1.7.0.0 --ghc 9.2.1 --cabal-project cabal.project
 ```
 
 Check `ghcup compile hls --help` for a full list of compilation options.
 
 ## Installation from source
 
-Direct installation from Source, while possible via `cabal install haskell-language-server`
+Direct installation from source, while possible via `cabal install haskell-language-server`
 and `stack install --stack-yaml stack-<GHCVER>.yaml`, is not recommended for most people.
-Said command builds the `haskell-language-server` binary and installs it in the default Cabal binaries folder,
+Said command builds the `haskell-language-server` binary and installs it in the default `cabal` binaries folder,
 but the binary will only work with projects that use the same GHC version that built it.
```

Comment:
Can we somehow highlight this line a little?

I personally had missed this while skimming through the docs and realized it later on. I think adding it as a note maybe can highlight the point.

Reply:
I think this section shouldn't be emphasized anyway and there is a section about this on the troubleshooting page already (which hopefully people actually read...).

Reply:
Generally there is a lot of stuff in the installation doc, and I wonder if it could be organized better :thinking: